quinn/src/runtime.rs

```diff
@@ -58,6 +54,16 @@ pub trait AsyncUdpSocket: Send + Debug + 'static {
     fn may_fragment(&self) -> bool {
         true
     }
+
+    /// Maximum number of datagrams that a [`Transmit`] may encode
```

Comment:
Style nit: should these go before or after `may_fragment()`?

Reply:
As far as I'm concerned they're getters with morally similar roles so the ordering isn't significant. Do you have a preference?

Reply:
I'd judge getters that yield a number as trivially higher-complexity than a getter that yields `bool` so just for consistency I'd probably move these ones above `may_fragment()`. But that's definitely the nittest of nits.